bin/start.sh

```diff
@@ -5,6 +5,8 @@
 #
 # ATTENTION: You would want to use dev_start.sh script, while developing, instead.
 
+NB_IS_CONTAINER="${NB_IS_CONTAINER:-0}"
+
```

Comment:
Thank you. Isn't it safer to make it

```
export NB_IS_CONTAINER="${NB_IS_CONTAINER:-0}"
```

Reply:
Yes - If you need to read the value of NB_IS_CONTAINER from a subshell then export is what we need here if it's only the current shell declaring it is fine.